generator/src/FluentStageFactoryGenerator.php

```diff
@@ -45,10 +46,11 @@ private function createFluentFactoryTrait(GeneratorDefinition $definition): PhpN
         $namespace->addUse(self::FACTORY_CLASS);
         $namespace->addUse(StageInterface::class);
         $namespace->addUse(Pipeline::class);
+        $namespace->addUse(stdClass::class);
 
         $trait->addProperty('pipeline')
             ->setType('array')
-            ->setComment('@var list<StageInterface>')
+            ->setComment('@var list<StageInterface|array<string,mixed>|stdClass>')
```

Comment:
Why was `stdClass` added? In https://github.com/mongodb/laravel-mongodb/pull/2738/files#r1536149792, you only needed `array<string,mixed>`.

Reply:
Because that's what is accepted by the Pipeline class.

Reply:
An existing Pipeline object is also accepted by the Pipeline constructor (for appending IIRC). I just assumed this would be as restrictive as possible based on what the trait enables users to do.

Defer to you.

Reply:
The property is public. The user can append anything to the array.